src/mono/mono/mini/dwarfwriter.c

```diff
@@ -204,7 +204,7 @@ emit_escaped_string (MonoDwarfWriter *w, char *value)
 {
 	int i, len;
 
-	len = strlen (value);
+	len = (int)strlen (value);
```

Comment:
This pattern seems very common and it concerns me we are going to have to update this for other CA warnings. Many of them make sense but it seems like we are sort of hiding issues.